From: Johannes Thumshirn <jthumshirn@suse.de>
Date: Thu, 24 Mar 2016 12:12:59 +0100
Subject: kABI: protect enum scsi_target_state
Patch-mainline: Never, kabi

Protect enum scsi_target_state's STARGET_REMOVE by __GENKSYSMS__. This value
is only valid in the remove path of the SCSI target state machine.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 include/scsi/scsi_device.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/scsi/scsi_device.h
+++ b/include/scsi/scsi_device.h
@@ -254,7 +254,9 @@ enum scsi_target_state {
 	STARGET_CREATED = 1,
 	STARGET_RUNNING,
 	STARGET_DEL,
+#ifndef __GENKSYMS__
 	STARGET_REMOVE,
+#endif
 };
 
 /*
